packages/agents/src/AgentFunctionBase.ts

```diff
@@ -0,0 +1,21 @@
+import { Agent, AgentFunctionResult, AgentOutputType } from "@evo-ninja/agent-utils"
+import { ChatCompletionRequestMessage } from "openai";
+import { Result } from "@polywrap/result";
+
+export interface HandlerResult {
```

Comment:
This type is being defined twice. Also why don't we use the existing type that has `outputs` and `messages` defined?